bndtools.core/src/bndtools/launch/BndDependencySourceContainer.java

```diff
@@ -78,14 +75,8 @@ public ISourceContainerType getType() {
                             result.add(new JavaProjectSourceContainer(targetJavaProj));
                         }
                     } else if (runbundle.getType() == TYPE.REPO) {
-                        IPath bundlePath = Central.toPath(runbundle.getFile());
-                        if (bundlePath != null) {
-                            IFile bundleFile = ResourcesPlugin.getWorkspace().getRoot().getFile(bundlePath);
-                            if (bundleFile != null) {
-                                ArchiveSourceContainer tempArchiveCont = new ArchiveSourceContainer(bundleFile, false);
-                                result.add(tempArchiveCont);
-                            }
-                        }
+                        ExternalArchiveSourceContainer container = new ExternalArchiveSourceContainer(runbundle.getFile().toString(), false);
```

Comment:
So I believe if `IPath bundlePath` and `IFile bundleFile` is not null that means we have a workspace archive so we should use `ArchiveSourceContainer` as before.  But if those aren't in the workspace we could fallback to using the `ExternalArchiveContainer`.